webpack-common.js

```diff
@@ -80,7 +81,8 @@ export function getRules({ babelQuery, bundleStylesWithJs = false } = {}) {
             },
           ],
         }),
-      }, {
+      },
+      {
```

Comment:
is this really needed? 😛

Reply:
It's easier to read! But it was from a patch where I did more to the webpack config... now it's all that remains.